### ISSUE TITLE
{Freeway-Bug}[AdaptiveCards- Product Video]: Captions or transcription is not available while playing the pre-recorded video.

### ISSUE THREAD

**Issue by vagpt** [microsoft/AdaptiveCards#8177]
### Target Platforms

Other

### SDK Version

1.5

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
URL: [[https://adaptivecards.io/](https://nam06.safelinks.protection.outlook.com/?url=https://adaptivecards.io/&data=04%7c01%7cv-ibalam%40microsoft.com%7cbd35d875ec3f4539762308d88f709775%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637417061036711036%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=dUolQbKGi1J8gsoxC2JG5TGoNi41msPfIIjKSIyPGHM%3D&reserved=0)
](https://adaptivecards.io/samples/ProductVideo.html)
OS Version: 22H2(OS Build 25247.1000)
Browser Version: Version 110.0.1543.0 (Official build) dev (64-bit)
Screen Reader: Narrator

**Repro Steps:**
1. Open the above URL in the browser.
2. Navigate to 'Product Video' and press enter key to activate it.
3. Navigate to product video sample and press enter key to activate it.
4. Navigate the video control and observe the issue.

**Actual Result:**
Captions or transcription options are not present in the cards under the product video. 

**Expected Result:**
Captions or transcription options should be available for the video under product video.

**User Experience:**
A person who can't listen to the voice after playing the video will find difficulty understanding voice if 'Live caption' and Transcript' will not be available.

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/s/accessibility/EVcPxC3kj3ZPhu1c2kGo9PoBEo0SF5vzBMaPBgVs2Sq6tw?e=zR44ln

"Have feedback to share on Bugs ? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-apoorva%40microsoft.com%7c551850210d444be9b28308d9691e3dde%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656400998658942%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=l6w0SvVcMSvyGnJ6%2B6uSSgd3braF2dZ7/AdBrqqzIR8%3D&reserved=0) “.

### Screenshots

https://user-images.githubusercontent.com/95913748/208825626-675f4de0-5382-4485-ae6f-9c683a37d844.mp4

### Card JSON

```JSON
NA
```

**Comment by anna-dingler** [comment]
Hello @vagpt,

I'm closing this bug because it was resolved in https://github.com/microsoft/AdaptiveCards/pull/8045. Captions for this video can be seen on [our test site](https://adaptivecardsci.z5.web.core.windows.net/samples/ProductVideo.html).

Thanks,
Anna

**Comment by vagpt** [comment]
@anna-dingler,

Could you please let us know by when the changes will be merged to the production site? As of now this issue is still reproducible on the production URL.

**Comment by vagpt** [comment]
@anna-dingler,

Could you please let us know by when the changes will be merged to the production site? As of now this issue is still reproducible on the production URL.

**Comment by anna-dingler** [comment]
Hi @vagpt,

We are planning on updating our production site by the end of February. I am closing this issue for now. Once new version is available, if the issue persists please open a new issue to track.

**Comment by vagpt** [comment]
Hi @anna-dingler,

Since this issue is still repro'ing hence we have created a new bug and can be tracked using below URL.

https://github.com/microsoft/AdaptiveCards/issues/8366
